### PR TITLE
Elaborated documentation regarding BasicAuth

### PR DESCRIPTION
**Pull request by RyanW8**
Elaborated the documentation on authenticating against a Cortex cluster using Basic Auth

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,14 @@ This action is configured using environment variables defined in the workflow. T
 | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- | ------- |
 | `CORTEX_ADDRESS`   | URL address for the target Cortex cluster                                                                                                                                                                                                  | `false`  | N/A     |
 | `CORTEX_TENANT_ID` | ID for the desired tenant in the target Cortex cluster                                                                                                                                                                                     | `false`  | N/A     |
```
Comment (gotjosh) [inline README.md:12]:
```suggestion
| `CORTEX_TENANT_ID` | ID for the desired tenant in the target Cortex cluster. Used as the username under HTTP Basic authentication.                                                                                                                                                                                     | `false`  | N/A     |
```

File: README.md [hunk 2]
```diff
@@ -10,10 +10,14 @@ This action is configured using environment variables defined in the workflow. T
...
-| `CORTEX_API_KEY`   | Optional password that is required for password protected Cortex clusters. An encrypted [github secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets ) is recommended. | `false`  | N/A     |
+| `CORTEX_API_KEY`   | Optional password that is required for password protected Cortex clusters. An encrypted [github secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets ) is recommended. Will use the `CORTEX_TENANT_ID` as a username | `false`  | N/A     |
```
Comment (gotjosh) [inline README.md:13]:
```suggestion
| `CORTEX_API_KEY`   | Optional password that is required for password-protected Cortex clusters. An encrypted [github secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets ) is recommended. Used as the password under HTTP Basic authentication. | `false`  | N/A     |
```

File: README.md [hunk 3]
```diff
@@ -10,10 +10,14 @@ This action is configured using environment variables defined in the workflow. T
...
 | `ACTION`           | Which action to take. One of `lint`, `prepare`, `check`, `diff` or `sync`                                                                                                                                                                  | `true`   | N/A     |
 | `RULES_DIR`        | Comma-separated list of directories to walk in order to source rules files                                                                                                                                                                 | `false`  | `./`    |
 
+## Authentication
+
+Currently cortextool only support basic authentication against a Cortex cluster. To enable basic authentication on this action ensure that a user with the `CORTEX_TENANT_ID` variable username exists on the basic auth proxy and set the `CORTEX_API_KEY` as the password.
```
Comment (gotjosh) [inline README.md:19]:
```suggestion
This GitHub Action uses [`cortextool`](https://github.com/grafana/cortex-tools) under the hood, `cortextool` uses HTTP Basic authentication against a Cortex cluster. The variable `CORTEX_TENANT_ID` is used as the username and `CORTEX_API_KEY` as the password.
```

**Review by gotjosh — APPROVED** [review]
Thank you very much for your contribution Ryan! I've left some comments.

**Comment by RyanW8** [comment]
> Thank you very much for your contribution Ryan! I've left some comments.

Cool, I've commited your recommendations.

**Pull request merged**